server/routes/admin.js

```diff
@@ -2,12 +2,22 @@ var express = require('express');
 var router = express.Router();
 const categoryClient = require('../dbClients/categoriesDB');
 const articleClient = require('../dbClients/articlesDB')
+const categoryDB = require("../dbClients/categoriesDB");
```

Comment:
why are you importing categoryDB - you can just use categoryClient (it's the same module).

Reply:
@kharazzoro did you read this comment